vm/loader/src/uefi/config.rs

```diff
@@ -179,6 +179,10 @@ pub enum BlobStructureType {
     Aspt = 0x21,
     Pptt = 0x22,
     Gic = 0x23,
+    Mcfg = 0x24,
```

Comment:
This feels shortsighted. Can we just make a generic table blob?

Reply:
That makes sense to me, but I think these changes have already been made on the UEFI/hyper-V side of things

Reply:
yes, we'll need additional changes on the UEFI side.  When available, you'll be able to pass AcpiTable multiple times